pandas/tests/io/test_clipboard.py

```diff
@@ -60,6 +60,9 @@ def setup_class(cls):
         # unicode round trip test for GH 13747, GH 12529
         cls.data['utf8'] = pd.DataFrame({'a': ['µasd', 'Ωœ∑´'],
                                          'b': ['øπ∆˚¬', 'œ∑´®']})
+        # Test for quotes and common delimiters in text
+        cls.data['delim_symbols'] = pd.DataFrame({'a': ['"a,\t"b|c', 'd\tef´'],
```

Comment:
What's the point of this?

Reply:
Excel can be very picky about how it handles copying/pasting text that includes delimiters or quote characters.  This dataframe caused some of the original clipboard tests to fail until I changed the to_clipboard and read_clipboard functions.